### PR TITLE
Course7

### PR DESCRIPTION
**Pull request by AryavT**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
React tutorial in nepali language

### Why is this valuable (or not)?
this is important so everybody with different languages such as nepali can learn code

### How do we know it's really free?
it is on youtube
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by sohansai — COMMENTED** [review]
We prefer 'playlist' links for YouTube contributions

**Comment by sohansai** [comment]
Also, please keep changes on this PR. don't close it and start a new one. It's not respectful to the reviewers because it erases the reviews.

**Comment by AryavT** [comment]
> Also, please keep changes on this PR. don't close it and start a new one. It's not respectful to the reviewers because it erases the reviews.

fixed it sorry about the issues

File: courses/free-courses-ne.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #### React
 
 * [React.js](https://www.youtube.com/playlist?list=PLckS_N3kOwFHhFEmcRs8jvX7xFaRFI4H1) - EverydayKarma 🇳🇵
+* [React Tutorial in Nepali](https://www.youtube.com/playlist?list=PLO5XYuYl2Cf8ebEJE-cCLiTaKj_3aPnTa) - CodeWithDinesh
```
Comment (eshellman) [inline courses/free-courses-ne.md:26]: alphabetization

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -274,6 +274,7 @@ Online School
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
 * [Python Bangla Tutorials for Beginners](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rz5XqMsTk41_j15_6682BN) - Anisul Islam
+* [Python Bangla Tutorial & Data Science Full Course || Full Guidelines for Python, ML, AI, DL & Django](https://www.youtube.com/playlist?list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ) - Study Mart
```
Comment (eshellman) [inline courses/free-courses-bn.md:277]: alphabetization

**Pull request closed without merging**